hack/upstream-install.sh

```diff
@@ -0,0 +1,15 @@
+#! /bin/bash
+
+ROOT_DIR=$(dirname "${BASH_SOURCE}")/..
+source "${ROOT_DIR}/hack/common.sh"
+source "${ROOT_DIR}/hack/lib/customize-manifests.sh"
+
+set +u
```

Comment:
I can remove this if I don't need to check the value of the `DISABLE_OCP_FEATURES` environment variable.

Reply:
We shouldn't ever need to check `DISABLE_OCP_FEATURES` in these scripts unless it's for running local. It shouldb ehandled by the manifests themselves.